packages/fluentui/react-northstar/src/themes/teams/componentStyles.ts

```diff
@@ -18,6 +18,9 @@ export { buttonStyles as Button } from './components/Button/buttonStyles';
 export { buttonGroupStyles as ButtonGroup } from './components/Button/buttonGroupStyles';
 export { buttonContentStyles as ButtonContent } from './components/Button/buttonContentStyles';
 
+export { breadcrumbItemStyles as BreadcrumbItem } from './components/Breadcrumb/breadcrumbItemStyles';
```

Comment:
Will we have any styles for `Breadcrumb`?

Reply:
@assuncaocharles

Reply:
I would guess so, very simple one. Since we are using `div` for `Breadcrumb.Item` I guess we need `displa: inline-block` there. Do you disagree @layershifter ?